cmd/manager/main.go

```diff
@@ -203,7 +214,7 @@ func parseConfiguration(log *zap.SugaredLogger) Config {
 	config.WatchedNamespaces = make(map[string]bool)
 	for _, namespace := range strings.Split(watchedNamespace, ",") {
 		namespace = strings.TrimSpace(namespace)
-		log.Infof("The Operator is watching the namespace %s", namespace)
+		fmt.Printf("The Operator is watching the namespace %s\r\n", namespace)
```

Comment:
Can this be done through the logger instead of printf? If so it will be more consistent.

Reply:
Remove this one. Added config printing when application starts.